doc/source/whats_new.rst

```diff
@@ -137,13 +139,13 @@ API
    - find_events and read_events functions have a new parameter `mask` to set some bits to a don't care state by `Teon Brooks`_
 
    - New channels module including layouts, electrode montages, and neighbor definitions of sensors which deprecates
-    ``mne.layouts`` by `Denis Engemann`_
+    `mne.layouts` by `Denis Engemann`_
```

Comment:
I'm using OSX. I dled the package and installed it but pygraphviz just yelled a lot. I tried this command from the source folder:
```
sphinx-build -D plot_gallery=0 -D make_flow_diagram=0 -b html . build/html
```
but it errors (can't find the helmet images because it is looking for it here: 'source/_images/mne_helmet.png'). I tried to move up a level to try to get the right location for the helmet but it yells and says that the conf.py isn't in that directory